lib/src/gainmapmath.cpp

```diff
@@ -139,12 +150,13 @@ Color srgbOetf(Color e) { return {{{srgbOetf(e.r), srgbOetf(e.g), srgbOetf(e.b)}
 ////////////////////////////////////////////////////////////////////////////////
 // Display-P3 transformations
 
-// See SMPTE EG 432-1, Equation 7-8.
-static const float kP3R = 0.20949f, kP3G = 0.72160f, kP3B = 0.06891f;
+// Display P3 Luminance
+// See, https://registry.khronos.org/DataFormat/specs/1.3/dataformat.1.3.html#PRIMARIES_DISPLAYP3
+static const float kP3R = 0.2289746f, kP3G = 0.6917385f, kP3B = 0.0792869f;
```

Comment:
This link shows the equation with D65 white point, however we are using D50. The original parameter is right.

Reply:
The numbers are correct. Keep the original comment, but change the equation 
```
// See SMPTE EG 432-1, Equation G-7.
```